src/main.rs

```diff
@@ -65,8 +74,29 @@ fn main() {
     gtk::init().unwrap();
     env_logger::init();
 
-    let res = gio::Resource::load(config::RESOURCES_FILE).expect("Could not load gresource file");
-    gio::resources_register(&res);
+    let resources = match env::var("MESON_DEVENV") {
+        Err(_) => gio::Resource::load(config::RESOURCES_FILE.to_owned())
+            .expect("Unable to load resources.gresource"),
+        Ok(_) => match env::current_exe() {
+            Ok(path) => {
+                let mut resource_path = path;
```

Comment:
You can directly match `Ok(mut resource_path)`. This middle variable is unnecessary

Reply:
Ah true, thanks for noticing that